webpack/scenes/Subscriptions/Manifest/SCAContainer.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useRef } from 'react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import PropTypes from 'prop-types';
+import SimpleContentAccess from './SimpleContentAccess';
+
+const SCAContainer = (props) => {
+  const {
+    isSimpleContentAccessEnabled,
+  } = props;
+  const [enabled, setEnabled] = useState(isSimpleContentAccessEnabled);
+  const hiddenRailsInput = useRef(document.querySelector('#content_access_mode'));
+
+  const infoMsgOverride = enabledVal => (enabledVal !== isSimpleContentAccessEnabled) || undefined;
```

Comment:
I should probably add comments here, but a note about how this works:
* The `isSimpleContentAccessEnabled` prop is set by the Rails view in `_edit_override.html.erb` and sets the initial state of the toggle switch.
* The Rails view also creates a hidden `<input>` element which is accessed here via `useRef`.  Its value is kept in sync with the state of the switch.  This is so it can eventually end up as `params[:content_access_mode]` when the form is submitted.
* `infoMsgOverride` is a function that decides what the text below the toggle should be.  If the user is about to change the content access mode, it displays the informative messages below; if not, the `undefined` value triggers the defaultProp in `<SimpleContentAccess />`.